homework/max-of-vector/maxOfVector.hpp

```diff
@@ -1,8 +1,16 @@
 #pragma once
 #include <vector>
 #include <limits>
+#include <algorithm>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
-}
+    auto max = vec[0];
+    for(const auto& el:vec)
+    {
+        if(max<el)
+        {
+            max = el;
+        }
+    }
+    return max;
+}
```

Comment:
Newline needed at the end

Reply:
Actually we don't need it in modern compiler. This is legacy form C.